pom.xml

```diff
@@ -118,7 +118,7 @@
     <nuxeo.features.version>10.10-HF05-SNAPSHOT</nuxeo.features.version>
     <nuxeo.addons.core.version>10.10-HF05-SNAPSHOT</nuxeo.addons.core.version>
 
-    <nuxeo.connect.client.version>1.7.5</nuxeo.connect.client.version>
+    <nuxeo.connect.client.version>1.7.6-SNAPSHOT</nuxeo.connect.client.version>
```

Comment:
isn't there an associated change in `maven-enforcer-plugin` exclusions?

Reply:
For me it's only in the release profile and there is https://jira.nuxeo.com/browse/NXP-27190 to do before HF05 release